test/src/unit-udt_macro.cpp

```diff
@@ -128,3 +210,69 @@ TEST_CASE_TEMPLATE("Serialization/deserialization via NLOHMANN_DEFINE_TYPE_INTRU
         CHECK_THROWS_WITH_AS(p3 = json(j), "[json.exception.out_of_range.403] key 'age' not found", json::out_of_range);
     }
 }
+
+TEST_CASE_TEMPLATE("Serialization/deserialization of classes with 26 public/private member variables via NLOHMANN_DEFINE_TYPE_INTRUSIVE and NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE", T,
+                   persons::person_with_private_alphabet,
+                   persons::person_with_public_alphabet)
+{
+    SECTION("alphabet")
+    {
+        {
+            T obj1;
+            nlohmann::json j = obj1; //via json object
+            T obj2;
+            j.get_to(obj2);
+            assert(obj1 == obj2);
```

Comment:
Please use a doctest construct for this and the other `assert` usages.

Reply:
So tried ```CHECK(obj1 == obj2)``` and didn't work. So replaced with ```bool ok = (obj1 == obj2); CHECK(ok);```. Don't know why the first version didn't work (I'm new to doctest) but the test is the same.

Reply:
I think doctest needs a way to produce a string from an object in case something goes wrong. The best would be the following:

```cpp
CHECK((obj1 == obj2))
```

(note the double parentheses)